WordPress/gutenberg
# Mobile - Columns block - Fix transforming into a Group block crash

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/6100

**Related PRs:**
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6129

## What?
This PR fixes a crash when transforming a nested Columns block into a Group block.

By checking the error it was happening around:

```
Symbolicated stack trace:

packages/block-library/src/column/edit.native.js:169:ColumnEdit
```

Checking the breadcrumbs shows the last UI change before the crash:

```
screen: UIAlertController, 
title: Transform Columns to, 
```

## Why?
To fix some crashes in production related to transforming Columns blocks into Group blocks.

## How?
By moving part of the logic of `renderAppender` within the `isSelected` if condition and also by adding a default value to `contentStyle` in case of `undefined` values.

## Testing Instructions

**Precondition:** Use the following HTML code:

```

<div class="wp-block-group">
<div class="wp-block-columns">
<div class="wp-block-column" style="flex-basis:100%">
<p></p>
</div>
</div>
</div>

```

- Open the post with that content
- Select the Columns block
- Open the block actions menu (three dots button in the Toolbar)
- Tap on `Transform block...`
- Tap on `Group`
- **Expect** the block to be transformed without crashing

## Screenshots or screencast 
Before|After
-|-
<video src="https://github.com/WordPress/gutenberg/assets/4885740/a4c2f179-9d13-4dd2-b6aa-8d523d1a742a" />|<video src="https://github.com/WordPress/gutenberg/assets/4885740/c632d200-8fb8-4c09-bc44-3a3f640fbaf0" />
